client/src/components/ItineraryList/InterchangeInfo.tsx

```diff
@@ -0,0 +1,22 @@
+import logo from '../../static/img/stay-seated.svg';
+import { Leg } from '../../gql/graphql.ts';
+
+/**
+ * Displays an icon if a leg has a stay-seated transfer from the previous one.
+ */
+export function InterchangeInfo({ leg }: { leg: Leg }) {
+  if (leg.interchangeFrom?.staySeated) {
+    return (
+      <img
+        alt="Stay-seated transfer"
+        title="Stay-seated transfer"
+        src={logo}
+        width="20"
+        height="20"
+        className="d-inline-block align-middle"
+      />
+    );
+  } else {
+    return <></>;
```

Comment:
This is equivalent to returning null, which is a more common pattern.

Reply:
I changed it to return null.